server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/project/ProjectService.java

```diff
@@ -193,6 +193,14 @@ public Project setArchived(Long id, @NotNull SetArchivedReq req) {
         return entityToModel(saved);
     }
 
+    @Transactional(rollbackFor = Exception.class)
+    @SkipAuthorize("permission check inside")
+    public List<Project> listAll(Boolean archived) {
```

Comment:
In the scenario of applying for project permissions, it is more reasonable to only return limited project information. For example, filtering out the project members' info.

Reply:
p.s. Why do we need the 'archive' parameters? It seems like it's not necessary to apply for archived project permissions.

Reply:
For Q1, this api originally only returns basic information about the project.
For Q2, I thought that if there is no `archive` param, how does the front-end know if it contains archived items?